tracer/test/benchmarks/Benchmarks.Trace/ActivityBenchmark.cs

```diff
@@ -0,0 +1,265 @@
+﻿#nullable enable
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using BenchmarkDotNet.Attributes;
+using Datadog.Trace;
+using Datadog.Trace.Activity.DuckTypes;
+using Datadog.Trace.Activity.Handlers;
+using Datadog.Trace.Configuration;
+using Datadog.Trace.DuckTyping;
+using ActivityIdFormat = System.Diagnostics.ActivityIdFormat;
+using ActivityKind = Datadog.Trace.Activity.DuckTypes.ActivityKind;
+using ActivityListener = System.Diagnostics.ActivityListener;
+using ActivitySamplingResult = System.Diagnostics.ActivitySamplingResult;
+using ActivitySource = System.Diagnostics.ActivitySource;
+using ActivityStatusCode = Datadog.Trace.Activity.DuckTypes.ActivityStatusCode;
+
+namespace Benchmarks.Trace;
+
+[MemoryDiagnoser]
+[BenchmarkAgent6]
+public class ActivityBenchmark
+{
+    private const string SourceName = "BenchmarkSource";
+    private static readonly Datadog.Trace.Activity.DuckTypes.ActivitySource _duckSource;
+    private static readonly DateTime _startTime = DateTimeOffset.FromUnixTimeSeconds(0).UtcDateTime;
+    private static readonly DateTime _endTime = DateTimeOffset.FromUnixTimeSeconds(5).UtcDateTime;
+
+    private static readonly ActivitySource _source;
+
+    static ActivityBenchmark()
+    {
+        _source = new ActivitySource(SourceName);
+
+        var activityListener = new ActivityListener { ShouldListenTo = _ => true, Sample = (ref ActivityCreationOptions<ActivityContext> options) => ActivitySamplingResult.AllData };
+
+        ActivitySource.AddActivityListener(activityListener);
+
+        var settings = new TracerSettings { StartupDiagnosticLogEnabled = false, TraceEnabled = false};
+
+        Tracer.UnsafeSetTracerInstance(new Tracer(settings, new DummyAgentWriter(), null, null, null));
```

Comment:
I'm just wondering - are these necessary 🤔 We're not doing any manual or auto instrumentation here are we?

Reply:
Nope, I think they were originally and I just didn't remove them, removed them though now thanks!